src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+
+import java.util.*;
```

Comment:
Try to avoid wildcard imports and instead import individual classes explicitly as per the coding standard.